### PR TITLE
Email Integrations Enhancement - Categories

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5350)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Screenshot 2023-05-01 at 10 21 35

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json [hunk 1]
```diff
@@ -180,7 +180,16 @@
...
+                            },
+							{
+								"endCol": 2,
+								"fieldId": "categories",
+								"height": 22,
+								"id": "5e2287d0-e502-11ed-ba0f-99a713ead7dc",
+								"index": 11,
+								"sectionItemType": "field",
+								"startCol": 0
+							}
```
Comment (ArikDay) [inline Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:192]: Make it align and on the same line like the rest of the fields

**Review by ArikDay — CHANGES_REQUESTED** [review]

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 1 above)
Comment (MLainer1) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:192]: Screenshot 2023-05-01 at 11 42 27
In the code itself it looks ok, maybe the display in github making it look weird

**Review by ArikDay — APPROVED** [review]

File: Packs/EmailCommunication/ReleaseNotes/2_0_7.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Mappers
+##### MS Graph Mail Single User - Incoming Mapper - Email Communication
+- Updated the **fetch-incidents** command to include *Categories*.
+
+##### MS Graph Mail - Incoming Mapper - Email Communication
+- Updated mapper to include *Categories*.
```
Comment (dantavori) [inline Packs/EmailCommunication/ReleaseNotes/2_0_7.md:7]:
```suggestion
##### MS Graph Mail Single User - Incoming Mapper - Email Communication
- Added the *Categories* field mapping for the *Phishing* and *Email Communication* incident types.

##### MS Graph Mail - Incoming Mapper - Email Communication
- Added the *Categories* field mapping for the *Phishing* and *Email Communication* incident types.
```

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_4_19.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+##### Microsoft Graph Mail Single User
+- Updated the Docker image to: *demisto/crypto:1.0.0.56396*.
+- Updated the **fetch-incidents** command to include *Categories*.
+
+#### Mappers
+##### Microsoft Graph Mail Single User - Incoming Mapper
+- Updated mapper to include *Categories*.
+
+##### Microsoft Graph Mail Mapper
+- Updated mapper to include *Categories*.
```
Comment (dantavori) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_4_19.md:17]:
```suggestion
- Updated the ***fetch-incidents*** command by including email categories as part of the retrieved incidents.

##### Microsoft Graph Mail Single User
- Updated the Docker image to: *demisto/crypto:1.0.0.56396*.
- Updated the ***fetch-incidents*** command by including email categories as part of the retrieved incidents.

#### Mappers
##### Microsoft Graph Mail Single User - Incoming Mapper
- Added the *Categories* field mapping for *Phishing* incident type.

##### Microsoft Graph Mail Mapper
- Added the *Categories* field mapping for *Phishing* incident type.
```

**Review by dantavori — APPROVED** [review]
let's schedule a quick demo for this

**Pull request merged**